string-manipulation-3/src/title-case.js

```diff
@@ -0,0 +1,84 @@
+/* exported titleCase */
+
+/*
+- create an array of minor words
+- create variable for correct JavaScript capitalized
+- create variable for lower case JavaScript for condition
+- create variable for correct API capitalized
+- create variable for lower case API for condition
+- need to find out if individual word ends with colon so the next adjacent word will always get capitalized
+------------------
+- assuming input title letters are all lower case -- wrong, need to lowercase input sentence
+- Split input title with space to into a new array
+*/
+
+var minorWords = ['and', 'or', 'nor', 'but', 'a', 'an', 'the', 'as', 'at', 'by', 'for', 'in', 'of', 'on', 'per', 'to'];
+
+function titleCase(title) {
+  var titleArray = title.toLowerCase().split(' ');
+
+  var isMinorWordIndexes = [];
+  var isWordWithColonIndexes = [];
+  var isJavaScriptColon = [];
+  var isHyphenWordIndexes = [];
+  for (var i = 0; i < titleArray.length; i++) {
+    // check for minor word and append index at which minor is found
+    if (Array.prototype.includes.call(minorWords, titleArray[i])) {
```

Comment:
What's with all the usage of `.call`?

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes